### ISSUE TITLE
Assertion error when reading specific file

### ISSUE THREAD

**Issue by fratrik** [felipenoris/XLSX.jl#117]
I get an assertion error at this line:

https://github.com/felipenoris/XLSX.jl/blob/master/src/stream.jl#L147

when trying to run readxslx on this file.
[noutput_first_second_third.xlsx](https://github.com/felipenoris/XLSX.jl/files/3677229/noutput_first_second_third.xlsx)

Debugging shows me that the nodetype and nodename are actionally

```julia
debug> EzXML.nodetype(reader)
READER_SIGNIFICANT_WHITESPACE

debug> EzXML.nodename(reader)
"#text"
```

I wasn't sure what to investigate next.

**Comment by felipenoris** [comment]
Thanks! I'll take a look.

**Comment by felipenoris** [comment]
Fixed by 0b13cf5dfebe4ed9eda85b6e11bf4e6a2be6679a
